src/ServiceControl.AcceptanceTests/CustomChecks/Custom_check_transition_should_trigger_signalr_event.cs

```diff
@@ -36,7 +37,6 @@ public class MyContext : ScenarioContext
             public bool SignalrEventReceived { get; set; }
             public int SCPort { get; set; }
             public string SignalrData { get; set; }
-            public bool SignalRConnected { get; set; }
```

Comment:
No reason, it was a field that we don't use, we can put it back for diagnostics if we need it again